parser/Builder.cc

```diff
@@ -305,10 +306,10 @@ class Builder::Impl {
 
     unique_ptr<Node> assignable(unique_ptr<Node> node) {
         if (auto *id = parser::cast_node<Ident>(node.get())) {
-            auto name_str = id->name.show(gs_);
-            checkReservedForNumberedParameters(name_str, id->loc);
+            auto name = id->name.shortName(gs_);
```

Comment:
In `accessible` you mentioned an `ENFORCE(id->name.kind() == core::NameKind::UTF8)` that justifies use of `shortName` instead of `show` here. Do we need to check that here?

Reply:
It turns out that `Ident` are always constructed with `NameKind::UTF8` `NameRef`s.  I could add some logic to `Ident`'s constructor to verify this at construction time as a follow-up if you like?

Reply:
That sounds like a good idea if you happen to get a chance.